apps/src/lab2/levelEditors/levelData/EditToolboxBlocks.tsx

```diff
@@ -0,0 +1,167 @@
+import classNames from 'classnames';
+import {isEqual} from 'lodash';
+import React, {useCallback} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
+import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import {BlockTypes} from '@cdo/apps/music/blockly/blockTypes';
+import {
+  categoryTypeToLocalizedName,
+  dynamicCategories,
+} from '@cdo/apps/music/blockly/toolbox';
+import {defaultMaps} from '@cdo/apps/music/blockly/toolbox/definitions';
+import {
+  Category,
+  CategoryBlocksMap,
+  ToolboxType,
+} from '@cdo/apps/music/blockly/toolbox/types';
+import {BlockMode} from '@cdo/apps/music/constants';
+import {getTypedKeys, ValueOf} from '@cdo/apps/types/utils';
+
+import MultiCategorySelect from './MultiCategorySelect';
+
+import styles from './edit-music-level-data.module.scss';
+
+// Levelbuilder-facing English block names.
```

Comment:
Nit: I would think of these strings as "block text" rather than "block names". The latter feels more synonymous with types.

Reply:
Updated to "block text"